src/tir/transforms/lower_device_kernel_launch.cc

```diff
@@ -243,6 +275,7 @@ class DeviceKernelMutator : public StmtExprMutator {
   Optional<Target> current_target_;
   std::unordered_map<const GlobalVarNode*, KernelInfo> device_info_map_;
   std::unordered_set<const GlobalVarNode*> device_kernel_launch_;
+  std::unordered_set<const GlobalVarNode*> extern_method_call_;
```

Comment:
`method` -> `func`? Since we don't use that term.

Reply:
Good point, and renamed `extern_method_call_` to `extern_function_call_`.